include/core/HY_Features_MPI.hpp

```diff
@@ -29,6 +30,16 @@ namespace hy_features {
             return network.filter("cat");
         }
 
+        inline auto catchments(long lv) {
```

Comment:
Should this unused parameter be removed? It's not present in the non-MPI version.

Reply:
the non MPI version needs to be updated to have this signature also, Will try to fix that today.

Reply:
Resolved in b0d7395